### DISCUSSION TITLE
Using Plotly as backend

### DISCUSSION THREAD

**Discussion by bpajusco** [aesara-devs/aesara#1491] [Ideas]
Is there a plan to add Plotly as a backend for aesara chart templates?

**Comment by brandonwillard** [comment]
Do you mean this as an alternative to the [D3 and/or Dot charts](https://aesara.readthedocs.io/en/latest/troubleshoot/printing_drawing.html)?

**Comment by bpajusco** [comment]
Yes exactly, adding Plotly as backend on top of Boheh and Matplotlib

**Reply by brandonwillard** [reply]
Yeah, that would be cool.
